plugin/pkg/admission/resourcequota/resource_access.go

```diff
@@ -48,7 +48,7 @@ type quotaAccessor struct {
 	lister corelisters.ResourceQuotaLister
 
 	// liveLookups holds the last few live lookups we've done to help ammortize cost on repeated lookup failures.
-	// This let's us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
+	// This lets us handle the case of latent caches, by looking up actual results for a namespace on cache miss/no results.
```

Comment:
let's or let us

Reply:
> let's or let us

Haha, I can't aggre. `This lets us` is right.

Reply:
/ok-to-test